pkg/plugins/golang/v2/scaffolds/internal/templates/config/prometheus/monitor.go

```diff
@@ -53,6 +53,10 @@ spec:
   endpoints:
     - path: /metrics
       port: https
+      scheme: https
+      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      tlsConfig:
+        insecureSkipVerify: true
```

Comment:
At this point, no new features should go into go/v2
```suggestion
```

Reply:
As the current configuration doesn’t work, it could also be seen as a bug fix, but happy to remove this part.

Reply:
If that's true, then it can be left in I suppose.

Reply:
If it is a bugfix (haven't checked that) it should have the 🐛 (`:bug:`) emoji.

Reply:
I wasn't sure, but as the default configuration just doesn't work I think it makes sense.